homeassistant/components/weatherflow_cloud/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/weatherflow_cloud",
   "iot_class": "cloud_polling",
   "loggers": ["weatherflow4py"],
-  "requirements": ["weatherflow4py==0.2.23"]
```

Comment:
Can we bump in a separate PR?

Reply:
https://github.com/home-assistant/core/pull/125676

Done (although I guess I have to leave this change in for the time being... - can rebase after the other PR is accepted)